metrics/web/lib/common/presentation/app_bar/widget/metrics_app_bar.dart

```diff
@@ -32,6 +32,7 @@ class MetricsAppBar extends StatelessWidget {
                 lightAsset: 'icons/logo-metrics-appbar-light.svg',
                 width: 116.0,
                 height: 32.0,
+                fit: BoxFit.none,
```

Comment:
Can we avoid specifying this param here because this is a default value of `fit`?